.github/workflows/build-and-push-to-docker.yml

```diff
@@ -68,9 +73,15 @@ jobs:
           export LABEL_PREFIX=${{ matrix.BASE_IMAGE_TAG }}
         fi
 
-        docker build docker/py --tag $DOCKER_NAME-py:$LABEL_PREFIX-$LABEL --build-arg BASE_IMAGE="jupyter/base-notebook:${{ matrix.BASE_IMAGE_TAG }}"
-        echo "IMAGE_NAME=$DOCKER_NAME-py:$LABEL_PREFIX-$LABEL" >> $GITHUB_OUTPUT
-        
+        # needed by the makefile
+        export DOCKER_PREFIX="renku/renkulab"    
+        export DOCKER_LABEL="${LABEL_PREFIX}-${LABEL}"
+        # this is required by the makefile
+        export GIT_MASTER_HEAD_SHA=$(git rev-parse --short=7 --verify HEAD)
```

Comment:
can't this be shared by all steps somehow?

Reply:
This has changed a little; I figured that it could be possible to do this but I think I've already spent enough time on this for now.

Reply:
nonsense! 1 week minimum for PRs with with CI and docker builds